src/ui/EditTask.scss

```diff
@@ -1,5 +1,5 @@
 .tasks-modal {
-    > section {
+    section + section {
```

Comment:
For my own education, what does the `+` do?

I'm sure it's standard stuff, but it's not immediately obvious to me...

Reply:
Hope this example makes it clear =)

```html
<section>
    <...>
</section>
<section> <!-- apply style to this one since it comes after a section element -->
    <...> 
</section>
<section> <!-- and this one too =) -->
    <...>
</section>
<something>
<section> <!-- but not this one since there is a <something> element before -->
    <...>
</section>
```

Reply:
Not really TBH. How about adding a comment saying what the intention is?(

e.g. 'Do something to the second field, which is Description' or whatever...

I mean, it's not about documenting what the CSS means, it's about what the actual intention of that line is, expressed in terms of the fields in the modal...